File: src/main/java/com/natia/secretmod/hooks/GuiScreenHook.java
package com.natia.secretmod.hooks;

import com.natia.secretmod.features.MinionAnalyzer;
import net.minecraftforge.client.event.GuiScreenEvent;
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;

public class GuiScreenHook {

    @SubscribeEvent
    public void preBackground(GuiScreenEvent.BackgroundDrawnEvent event) {
        MinionAnalyzer.getInstance().backgroundDrawn(event);
    }
}

File: src/main/kotlin/natia/skytweaks/hooks/GuiScreenHook.kt
package natia.skytweaks.hooks

import natia.skytweaks.features.MinionAnalyzer
import net.minecraftforge.client.event.GuiScreenEvent
import net.minecraftforge.fml.common.eventhandler.SubscribeEvent

class GuiScreenHook {

    @SubscribeEvent
    fun preBackground(event: GuiScreenEvent.BackgroundDrawnEvent) {
        MinionAnalyzer.instance.backgroundDrawn(event)
    }
}
